LaTeX/syntax_test_latex.tex

```diff
@@ -42,6 +42,13 @@
 %             ^support.function.latex entity.name.newcommand.latex
 %                               ^ support.function.general.latex
 
+\newcolumntype{x}{>{$}c<{$}}
```

Comment:
You could also test, that `meta.function.newcolumntype.latex` stretches over the whole command.